content/docs/optimizing-performance.md

```diff
@@ -176,7 +176,11 @@ module.exports = {
 
 6. Події React будуть згруповані під міткою **User Timing**.
 
+<<<<<<< HEAD
```

Comment:
Replace URL in HEAD with https://calibreapp.com/blog/react-performance-profiling-optimization